doc/tutorial/index.rst

```diff
@@ -0,0 +1,198 @@
+.. _tutorial:
+
+===============
+Sphinx tutorial
+===============
+
+In this tutorial you will build a simple documentation project using Sphinx,
+and view it in your browser as HTML.
+The project will include narrative, handwritten documentation,
+as well as autogenerated API documentation.
+
+The tutorial is aimed towards Sphinx newcomers
+willing to learn the fundamentals of how projects are created and structured.
+You will create a fictional software library to generate random food recipes
+that will serve as a guide throughout the process,
+with the objective of properly documenting it.
+
+To showcase Sphinx automatic documentation generation capabilities
+you will use Python, which is the default :term:`domain`
```

Comment:
I think mentioning the default domain is not needed. It's not related to autodoc at all.

Reply:
Fair enough 👍🏽